packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/e2e/quanticFoldedResultList.e2e.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @lwc/lwc/no-document-query */
```

Comment:
I would add folding tests here. It just makes sense to me and we could re-use the data mocked in the cypress tests.
For the quantic results, you could add unit tests later there, but I do think it would be nice to test the behaviour or at least some sort of happy path with the folding here.

Reply:
But if we add this to these e2e tests it goes against our decisions on what to test as e2e.

There is no browser specific feature, no user interaction etc. that rely on folding. 

Maybe only the "Show More" when expanding a collection because it potentially triggers an analytics event?. But IMO it makes a ton more sense to be grouped wherever the actual collection is being displayed which is within the quanticResult and not within the result list itself.

Reply:
Even more, these child-results are actually even only rendered by the result-templates, so it's not even a behavior of the QuanticResult, but more of a behavior of the registered templates in the ResultTemplateManager.

I could add a e2e test to the FoldedResultList because it'd definitely be easier to test there, but it would go against our principles defined in our testing strategy to only e2e tests what we absolutely need to test via e2e. And it would also be an e2e test for the "wrong" component.